### PR TITLE
Circle CI: Test for configuration validity of bumpversion

### PR DESCRIPTION
**Pull request by axic**
@chfast not sure this is useful, please feel free to close.

### CODE REVIEW

**Comment by axic** [comment]
Also apparently `bumpversion` is not maintained, but a fork is: https://github.com/c4urself/bump2version

**Comment by chfast** [comment]
I noticed. The bumpversion from Ubuntu is newer from the one from the original repo. And it has additional features like signed annotated tags.

File: circle.yml [hunk 1]
```diff
@@ -17,6 +17,21 @@ jobs:
...
+    docker:
+    docker:
+    - image: ethereum/cpp-build-env:10
+    steps:
+    - checkout
+    - run:
+        name: "Check bumpversion"
+        command: |
+          pip3 install bumpversion
+          export PATH="/home/builder/.local/bin:$PATH"
```
Comment (chfast) [inline circle.yml]: Do it before `pip3 install`, otherwise `pip3` will complain.

File: circle.yml [hunk 2]
```diff
@@ -17,6 +17,21 @@ jobs:
...
+  bumpversion:
+    docker:
+    docker:
+    - image: ethereum/cpp-build-env:10
+    steps:
+    - checkout
+    - run:
+        name: "Check bumpversion"
+        command: |
+          pip3 install bumpversion
```
Comment (chfast) [inline circle.yml]: Don't you want `pip3 install --user`?

File: circle.yml [hunk 3]
```diff
@@ -17,6 +17,21 @@ jobs:
         command: |
           codespell --quiet-level=4 --ignore-words=./.codespell-whitelist
 
+  bumpversion:
```
Comment (chfast) [inline circle.yml]: Add it as a step in `lint` job because starting the `ethereum/cpp-build-env` container is expensive.

File: circle.yml (hunk 1 above)
Comment (axic) [reply circle.yml]: Changed.

File: circle.yml (hunk 2 above)
Comment (axic) [reply circle.yml]: Dunno, can try.

File: circle.yml (hunk 3 above)
Comment (axic) [reply circle.yml]: Changed.

File: circle.yml (hunk 2 above)
Comment (axic) [reply circle.yml]: That uses the same path, still needs the `export PATH`.

**Comment by axic** [comment]
@chfast ready to be merged

**Pull request merged**